tests/Install-Scripts.Test/GivenThatIWantToInstallDotnetFromAScript.cs

```diff
@@ -595,6 +595,24 @@ public void WhenFailingToInstallWithFeedCredentials(string? runtime, string chan
             commandResult.Should().NotHaveStdOutContainingIgnoreCase(feedCredential);
         }
 
+        [Theory]
+        [InlineData("7.0.1", Quality.Daily)]
+        [InlineData("6.0.1", Quality.Signed)]
+        public void WhenBothVersionAndQualityWereSpecified(string version, Quality quality)
```

Comment:
If we don't have already, I suggest adding cases where:
- Only quality is specified
- Only version is specified
- Neither is specified
and test that the error message is not displayed in all cases.
I suspect we already have these cases being used in the tests, but probably not all.

Reply:
Done, thank you !